# Pensamiento geométrico para visualizar y argumentar

# Clase 2

### Sectores circulares

El área de un sector circular es

$$ A= \dfrac{\theta R^2}{2} $$

El área de un sector circular es

$$ A= \dfrac{t\pi R^2}{360} $$

**Nota importante:** Aquí el ángulo $\theta$ se mide en radianes y el ángulo $t$ se mide en grados.

<img src="seg_circular1.png" align="middle" width="300">

El perímetro de un sector circular se mide como
$$ P=R(\theta+2) $$

El perímetro de un sector circular se mide como
$$ P=R\left(\dfrac{\pi t}{180}+2\right) $$

**Nota importante:** Aquí el ángulo $\theta$ se mide en radianes y el ángulo $t$ se mide en grados.

* ***Ejemplo:*** Calcular el área de un sector circular de radio 3 cm y ángulo de $30\,{}^\circ$.

<img src="seg_circular1.png" align="middle" width="300">

* Datos:
    * $R=3$
    * $t=30^\circ$
    * $\theta=\dfrac{\pi}{6}\ rad$
    * $A=?$

$$ 30^\circ=\dfrac{30\pi}{180}\ rad=\dfrac{\pi}{6}\ rad $$

* Fórmula:

$$ A=\dfrac{\theta R^2}{2} $$

* Sustitución:

$$ A=\dfrac{\frac{\pi}{6} (3)^2}{2} $$

$$ =\dfrac{\frac{\pi}{6} 9}{2} $$

$$ =\dfrac{\frac{9\pi}{6}}{2} $$

$$ =\dfrac{\frac{3\pi}{2}}{\frac{2}{1}} $$

$$ =\dfrac{3\pi}{4} $$

In [1]:
3π/4

2.356194490192345

El área es $\dfrac{3\pi}{4}\ \text{cm}^2=2.356\ \text{cm}^2$

* ***Ejemplo:*** Calcular el área y perímetro de un sector circular de radio 5 pulgadas (in) y ángulo de $\dfrac{3}{4}\pi\ \rm{rad}$.

* Datos:
    * $R=5$
    * $\theta=\dfrac{3}{4}\pi\ \rm{rad}$
    * $A=?$

* Fórmula:

$$ A=\dfrac{\theta R^2}{2} $$

* Sustitución:

$$ A=\dfrac{\frac{3}{4}\pi (5)^2}{2} $$

$$ =\dfrac{\frac{3}{4}\pi 25}{2} $$

$$ =\dfrac{\frac{3}{4}\pi \frac{25}{1}}{2} $$

$$ =\dfrac{\frac{75}{4}}{\frac{2}{1}}\pi $$

$$ =\dfrac{75}{8}\pi $$

El área es $\dfrac{75\pi}{8}\ \text{in}^2$

* ***Ejemplo:*** Calcular el perímetro de un sector circular de radio 5 pies (ft) y ángulo de $\dfrac{3}{4}\pi\ \rm{rad}$.

<img src="seg_circular1.png" align="middle" width="300">

* Datos:
    * $R=5$
    * $\theta=\dfrac{3}{4}\pi\ \rm{rad}$
    * $P=?$

* Fórmula:

$$ P=R(2+\theta) $$

* Sustitución:

$$ P=5\left(2+\dfrac{3}{4}\pi\right)=10+\dfrac{15}{4}\pi=21.78 $$

In [2]:
10+15π/4

21.780972450961723

El perímetro del sector circular es $\left(10+\dfrac{15}{4}\pi\right)\ \text{ft}=21.78\ \text{ft}$

* ***Ejemplo:*** Calcular el perímetro de un sector circular de radio 3 y ángulo de $30\,{}^\circ$.

* Datos:

* Fórmula:

* Sustitución:

<img src="seg_circular1.png" align="middle" width="200">

## Triángulo

<img src="triangulo.png" align="middle" width="300">

**¿Qué es un triángulo (también trigono)?**
* Una figura geométrica que tiene 3 ángulos.
* Una figura geométrica que tiene 3 vértices no colineales.
* Un *polígono* (figura geométrica que tiene muchos lados) que tiene 3 lados (o también llamados catetos). 

**Importante:** La suma de los ángulos internos de cualquier triángulo es un ángulo llano:

$$ A+B+C=180^\circ=\pi\ \rm{rad} $$

**¿Qué es el cateto opuesto?**

Imaginemos un cuarto que tiene forma de triángulo. Ponemos una linterna en el ángulo $A$. El *cateto opuesto* es la pared que se ilumina.

<img src="cateto_opuesto.png" align="middle" width="300">

Entonces, cuando escribimos un triángulo como en las dos figuras anteriores tenemos que:
* "$a$ es el cateto opuesto al ángulo $A$"  ó  "$A$ es el ángulo opuesto al cateto $a$"
* "$b$ es el cateto opuesto al ángulo $B$" ó "$B$ es el ángulo opuesto al cateto $b$"
* "$c$ es el cateto opuesto al ángulo $C$" ó "$C$ es el ángulo opuesto al cateto $c$"

### Tipos de triángulos

<img src="triangulo.png" align="middle" width="300">

i) **Equilátero:** Tres lados iguales ($a=b=c$) ó tres ángulos iguales ($A=B=C=60^\circ=\dfrac{\pi}{3}\ rad$)

ii) **Isósceles:** Dos lados iguales (p.ej. $a=c$) ó dos ángulos iguales (p.ej. $A=C$)

iii) **Escaleno:** Ningún lado igual ($a\neq b\neq c$) ó ningún ángulo igual ($A\neq B\neq C$)

<img src="tipos_triang1.png" align="middle" width="700">

i) **Rectángulo:** Uno de sus ángulos es recto (vale $90^\circ$ ó $\dfrac{\pi}{2}\ \rm{rad}$)

ii) **Oblicuángulo:** Ninguno de sus ángulos es recto

<img src="tipos_triang2.png" align="middle" width="500">

Noten que:
* Un triángulo rectángulo puede ser un triángulo isósceles.
* Un triángulo rectángulo puede ser un triángulo escaleno.
* Un triángulo rectángulo **NO** puede ser un triángulo equilátero.
* Un triángulo oblicuángulo puede ser un triángulo isósceles.
* Un triángulo oblicuángulo puede ser un triángulo escaleno.
* Un triángulo oblicuángulo puede ser un triángulo equilátero.

### Teorema de Tales y triángulos

<img src="tales.png" align="middle" width="300">

*Si tenemos tres rectas paralelas y estas cortan a dos rectas que no son paralelas se producen segmentos proporcionales:*

$$ \dfrac{\overline{AB}}{\overline{AC}}=\dfrac{\overline{DE}}{\overline{DF}} $$

También se cumple que

$$ \dfrac{\overline{AB}}{\overline{BC}}=\dfrac{\overline{DE}}{\overline{EF}} $$

### Semejanza

Dos cosas semejantes si son de la misma forma aunque tengan distinto tamaño.

<img src="minime.jpg" align="middle" width="300">

### Triángulos semejantes

Dos triángulos son semejantes si tienen la misma forma pero distinto tamaño. Dicho de otra forma, dos triángulos son semejantes si los triángulos tienen los mismo ángulos y los lados son proporcionales. La manera en que nos aseguramos que sean semejantes es:
* si tienen todos sus ángulos iguales entre sí
* si tienen dos ángulos iguales entre sí
* si tienen un ángulo igual entre sí y sus dos de sus lados son proporcionales con los lados correspondientes del otro triángulo
* si todos sus lados son proporcionales entre sí

### Ejemplos de semejanza de triángulos

* ***Ejemplo:***

<img src="triangulo.png" align="middle" width="300">

Pinto una línea paralela y genero dos triángulos:
* $\Delta ABC$
* $\Delta DBE$

<img src="triang_sem1.png" align="middle" width="400">

Los separo:

<img src="triang_sem2.png" align="middle" width="550">

**¿Qué podemos decir?**

$$ \dfrac{\overline{AB}}{\overline{BD}}=\dfrac{\overline{BC}}{\overline{BE}}=\dfrac{\overline{AC}}{\overline{DE}} $$

**¿Qué podemos decir también?**

$$ \dfrac{\overline{AC}}{\overline{AB}}=\dfrac{\overline{DE}}{\overline{BD}} $$

***Ejemplo:*** Pensemos que los triángulos semejantes son:

<img src="semejanza1.png" align="middle" width="500">

$$ \dfrac{x+1}{4}=\dfrac{12}{6} $$

$$ \dfrac{x+1}{4}=2 $$

$$ x+1=2\cdot4=8 $$

$$ x=7 $$

$$ \dfrac{y-1}{8}=\dfrac{6}{12} $$

$$ \dfrac{8}{y-1}=\dfrac{12}{6} $$

$$ \dfrac{8}{y-1}=2 $$

$$ 8=2(y-1) $$

$$ \dfrac{8}{2}=y-1 $$

$$ 4=y-1 $$

$$ 5=y $$

* ***Ejemplo:***

<img src="triang_sem3.png" align="middle" width="450">

Los separo:

<img src="triang_sem4.png" align="middle" width="500">

***Ejemplo:*** El gato de Leticia subió a la parte superior de un poste. Leticia puede ver a su gato reflejado en un charco, que se encuentra a $1.6\ \rm{m}$ de ella. Si la altura de sus ojos es de $1.44\ \rm{m}$ y el poste se encuentra a $5.6\ \rm{m}$ de ella, ¿qué tan alto es el poste?

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

## Triángulos rectángulos

### Teorema de Pitágoras

*La suma de los catetos al cuadrado es igual a la hipotenusa al cuadrado.*
$$ a^2+b^2=c^2 $$

## Funciones trigonométricas

Las *funciones trigonométricas* se definen (sólo para triángulos rectángulos) en términos de los catetos y la hipotenusa.

$$ \sin x=\dfrac{\text{c.o.}}{\text{hip.}} $$

$$ \cos x=\dfrac{\text{c.a.}}{\text{hip.}} $$

$$ \tan x=\dfrac{\text{c.o.}}{\text{c.a.}} $$

* c.o.: cateto opuesto
* c.a.: cateto adyacente
* hip.: hipotenusa

El *cateto adyacente* es el cateto que se encuentra de lado al ángulo en el que nos fijemos.

<img src="triangulo_rect.png" align="middle" width="300">

* ***Ejemplo:*** Obtener las funciones trigonométricas para un triángulo general

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

Para el ángulo $A$
* c.o.: $a$
* c.a.: $b$
* hip.: $c$

Para el ángulo $B$
* c.o.: $b$
* c.a.: $a$
* hip.: $c$

Entonces:

$$ \sin A=\dfrac{a}{c}\qquad\qquad\sin B=\dfrac{b}{c} $$

$$ \cos A=\dfrac{b}{c}\qquad\qquad \cos B=\dfrac{a}{c}  $$

$$ \tan A=\dfrac{a}{b}\qquad\qquad \tan B=\dfrac{b}{a} $$

**¿Cómo "resolver" un triángulo rectángulo?**

**Caso #1:** Si me dan un lado y un ángulo, y me piden encontrar otro lado:
1. Necesitamos tener a la mano los datos que nos dan y el dato que piden calcular.
2. Necesitamos identificar cuál de las funciones trigonométricas incluyen todos estos datos.
3. Despejamos el dato a calcular en la fórmula.
4. Sustituir para terminar el cálculo.

* ***Ejemplo:*** Tenemos un TR que tiene su cateto opuesto de $5\ \rm{cm}$ a un ángulo de $35^\circ$. ¿Cuál es la hipotenusa?

* Datos:

* Fórmula:

* Sustitución:

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

* ***Ejemplo:*** Tenemos un TR que tiene su hipotenusa de $7\ \rm{cm}$ y un ángulo de $43^\circ$. ¿Cuál es el cateto adyacente a ese ángulo? ¿Cuál es el cateto opuesto?

<img src="triangulo_rect.png" align="middle" width="250">

* Datos:

* Fórmula:

* Sustitución:

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

* ***Ejemplo:*** Tenemos un TR que tiene su cateto opuesto de $3\ \rm{cm}$ a un ángulo de $70^\circ$. ¿Cuál es el otro cateto y la hipotenusa?

* Datos:

* Fórmula:

* Sustitución:

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

**Caso #2:** Si me dan dos lados y me piden un ángulo
1. Necesitamos tener a la mano los datos que nos dan y el dato que piden calcular.
2. Necesitamos identificar cuál de las funciones trigonométricas incluyen todos estos datos.
3. Despejamos el dato a calcular en la fórmula usando las funciones trigonométicas inversas.
4. Sustituir para terminar el cálculo.

* ***Ejemplo:*** Tenemos un TR que tienen catetos con valores de $4$ y $5\ \rm{cm}$. ¿Cuáles son los ángulos del triángulo?

* Datos:

* Fórmula:

* Sustitución:

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

* ***Ejemplo:*** Tenemos un TR que tiene un cateto $6$ e hipotenusa $6\sqrt{2}\ \rm{cm}$. ¿Cuáles son los ángulos del triángulo?

* Datos:

* Fórmula:

* Sustitución:

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

* ***Ejemplo:*** Tenemos un TR que tienen catetos con valores de $5$ y $12\ \rm{cm}$. ¿Cuáles son los ángulos del triángulo?

* Datos:

* Fórmula:

* Sustitución:

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

**Caso #3:** Es un problema donde sólo nos dan un dato explícitamente, y otro dato no explícitamente.

* ***Ejemplo:*** Una persona ve la cima de una montaña cuando le apunta con un telescopio que forma $30^\circ$ con el piso. Se aleja $2\ \rm{km}$ y ahora apunta el telescopio de tal manera que mantenga un ángulo de $25^\circ$ con el piso. Si la altura del telescopio respecto del piso es de $0.5\ \rm{m}$, ¿podemos saber la altura de la montaña?

<img src="montana.png" align="middle" width="900">

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

### La altura de cualquier triángulo

<img src="triangulo.png" align="middle" width="300">

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

### El área altura de cualquier triángulo

<img src="triangulo.png" align="middle" width="300">

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

## Triángulos oblicuángulos

### Ley de senos

Ésta nos sirve cuando:
* conocemos dos lados y un ángulo opuesto, y queremos conocer el otro ángulo opuesto
* conocemos dos ángulos y un lado opuesto, y queremos conocer el otro lado opuesto

$$ \dfrac{\sin A}{a}=\dfrac{\sin B}{b}=\dfrac{\sin C}{c} $$

<img src="triangulo.png" align="middle" width="300">

### Ley de cosenos

Ésta nos sirve cuando:
* conocemos los tres lados y queremos conocer el ángulo opuesto a cualquiera de los lados
* conocemos dos lados y el ángulo entre estos lados, y queremos conocer el lado opuesto a este ángulo.

*El cuadrado de un lado es igual a la diferencia de la suma de los cuadrados de los otros lados y el doble del producto de los otros lados con el coseno del ángulo opuesto al lado inicial.*

$$ a^2=b^2+c^2-2bc\cos A $$

$$ b^2=a^2+c^2-2ac\cos B $$

$$ c^2=a^2+b^2-2ab\cos C $$

<img src="triangulo.png" align="middle" width="300">

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $

$ $